tools/cldr-apps/js/src/views/DashboardPanel.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div id="dashboard">
+    <p v-if="fetchErr" class="st-sad">Error loading data: {{ fetchErr }}</p>
+    <p v-if="!data" class="st-sad">Loading…</p>
+    <div v-if="data && !fetchErr" id="scrollingwindow">
+        <div v-for="n in data.notifications" :key="n.notification">
+            <h3 class="collapse-review">
+                {{ n.notification }}
+            </h3>
+            <div v-for="g in n.entries" :key="g.header">
+                <h4>{{ g.section }}/{{ g.page }}/{{ g.header }}</h4>
+                <div v-for="e in g.entries" :key="e.xpath">
+                        <a v-bind:href="'#/'+[locale, g.page, e.xpath].join('/')">
+                    <span
+                        class="label label-info">
+                            {{ e.code }}
+                    </span> </a><br/>
+                    English: {{ e.english }}<br />
+                    Old: {{ e.old }}<br />
+                    Winning: {{ e.winning }}<br />
+                    <i span v-html="e.comment"></i><br />
+                    <hr />
+                </div>
+            </div>
+        </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  props: ["specialPage", "cldrOpts"],
+  data: function () {
+    return {
+      data: null,
+      fetchErr: null,
+      locale: null
+    };
+  },
+  created: function () {
+    this.fetchData();
+  },
+  methods: {
+    fetchData: function () {
+      const { cldrStatus, cldrSurvey } = this.cldrOpts;
+      const locale = cldrStatus.getCurrentLocale();
+      this.locale = locale;
+      const level = cldrSurvey.getSurveyUserCov();
+      const session = cldrStatus.getSessionId();
+      if (!locale) {
+        this.fetchErr = "Please choose a locale first.";
+        return;
+      }
+      fetch(`api/summary/dashboard/${locale}/${level}?session=${session}`)
+        .then((data) => data.json())
```

Comment:
let's consider the pros and cons of directing all requests thru cldrAjax

pros include the ability to do logging for requests uniformly, and the ability to "mock" the back end for testing

cldrAjax includes a queue, which is sometimes bypassed; it's not clear to me whether the queue is beneficial or why it was introduced in the first place

Reply:
As it's well supported I think we shoud switch to fetch.

cldrAjax was introduced because of browser incompatibilities. think that may be moot. 
The queue was probably due to back end inefficiencies.

Reply:
fetch can be mocked, https://github.com/wheresrhys/fetch-mock

Reply:
we can discuss further; good points about alternative available mocking methods and not reinventing the wheel! still I suspect we'd benefit from a cldrAjax.fetch even if it only called normal fetch plus optional logging

good to know about the queue -- let's remove the queue!